GliaWidgets/Sources/EntryWidget/EntryWidget.swift

```diff
@@ -51,6 +52,13 @@ public final class EntryWidget: NSObject {
                 ongoingEngagement = engagement
                 handleQueuesMonitorUpdates(state: environment.queuesMonitor.state, unreadSecureMessagesCount: unreadSecureMessageCount)
             }
+
+        engagementCancellable = environment.currentInteractor()?.$state
+            .sink { [weak self] engagement in
+                guard let self else { return }
+                interactorState = engagement
```

Comment:
Does it make sense if you rename `engagement` param with `state`?

Reply:
Yes good point. Fixed [2242fbb](https://github.com/salemove/ios-sdk-widgets/pull/1184/commits/2242fbb9a46c5feb768a65a9494eaf123da5dcf5)